contracts/FungibleToken.cdc

```diff
@@ -187,6 +186,7 @@ access(all) contract interface FungibleToken: ViewResolver {
                 //
                 self.balance == before(self.balance) - amount:
                     "New Vault balance must be the difference of the previous balance and the withdrawn Vault balance"
+                emit Withdrawn(type: result.getType().identifier, amount: amount, from: self.owner?.address, fromUUID: self.uuid, withdrawnUUID: result.uuid, balanceAfter: self.balance)
```

Comment:
With each parameter/argument this is getting harder to read.

Maybe chop down the argument list and put each argument on a separate line:

```suggestion
                emit Withdrawn(
                	type: result.getType().identifier,
                	amount: amount,
                	from: self.owner?.address,
                	fromUUID: self.uuid,
                	withdrawnUUID: result.uuid,
                	balanceAfter: self.balance
                )
```

Reply:
good suggestions! I'll fix those